src/org/labkey/test/tests/LinkedReportTest.java

```diff
@@ -40,12 +40,13 @@ public static void setupProject()
     /* Regression coverage : Issue 47004: Add Test Automation for LinkedReports  */
 
     @Test
-    public void testLinkedReportToExternalURL()
+    public void testLinkedReportToExternalURL() throws InterruptedException
     {
         goToProjectHome();
         goToManageViews().clickAddReport("Link Report");
         setFormElement(Locator.name("viewName"), REPORT_NAME);
         setFormElement(Locator.name("linkUrl"), LINK_REPORT_URL);
+        Thread.sleep(1000);  // Hack to prevent saving before the form is ready to submit, lacking a way to check before clicking
```

Comment:
`WebDriverWrapper` has a `sleep` method that wraps the `InterruptedException` automatically.

```suggestion
    public void testLinkedReportToExternalURL()
    {
        goToProjectHome();
        goToManageViews().clickAddReport("Link Report");
        setFormElement(Locator.name("viewName"), REPORT_NAME);
        setFormElement(Locator.name("linkUrl"), LINK_REPORT_URL);
        sleep(1000);  // Hack to prevent saving before the form is ready to submit, lacking a way to check before clicking
```

Reply:
Updated